books/projects/x86isa/machine/dispatch-macros.lisp

```diff
@@ -318,9 +320,11 @@
                       (eql proc-mode #.*compatibility-mode*)
                       (eql proc-mode #.*protected-mode*)))))
     :ud)
-   ((eq type-id :type-vex-gpr)
+   ((or (eq type-id :type-vex-gpr)
+        ;; BOZO Sol -- it seems like maybe this is a synonym?
+        (eq type-id :type-13))
     ;; from table 2.5.1 from volume 2.. we only need to additionally check
-    ;; the cpuid requirements:
+    ;; the cpuid requirements: (BOZO Sol -- maybe this is now table 2.6.1?)
```

Comment:
Yes, in the June 2024 version:
- All the 2.4.x sections in Volume 2 are now 2.5.x.
- All the 22-x tables in Volume 3 are now 23-x.